src/test/resources/net/sf/jabref/importer/fileformat/OvidImporterTestBib6.bib

```diff
@@ -1,6 +1,6 @@
 @Misc{,
   Title                    = {Correct positioning of percutaneous iliosacral screws with computer-navigated versus fluoroscopically guided surgery in traumatic pelvic ring fractures},
-  Author                   = {Verbeek, Jan MD and Hermans, Erik MD and van Vugt, Arie MD, PhD and Frolke, Jan Paul MD, PhD},
```

Comment:
Not obvious why they need to brag about their titles in the first place. :-) (And I know that some of you might end up to write Herr Prof. Dr. Ing. Hab. Prof. Dr.)